canonical-web-and-design/ubuntu.com
# Update UASD page

## Done

- Updates the UASD page as per [copydoc](https://docs.google.com/document/d/15URkXf2vh7bv89n-cWQPeaP2EehSNUnUCoZvGbNTYFA/edit#)
- Updates _TOC.html to accommodate new section

## QA

- Go to https://ubuntu-com-11406.demos.haus/legal/ubuntu-advantage-service-description and compare against [copydoc](https://docs.google.com/document/d/15URkXf2vh7bv89n-cWQPeaP2EehSNUnUCoZvGbNTYFA/edit#)
- Check the link to 'Embedded services' works as expected

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4978
